src/pages/notFoundPage.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import logo from '../assets/images/notfound.png';
+
+const NotFoundPage = () => (
+  <div className="flex items-center justify-center flex-col h-screen">
+    <img src={logo} alt="404 Page" className="sm:w-2/4 md:w-1/5" />
+    <h2 className="text-secondary my-6 md:w-2/5 text-center text-sm font-semibold">
+      THE PAGE YOU ARE LOOKING FOR MIGHT HAVE BEEN REMOVED HAD ITS NAME CHANGED
```

Comment:
THE PAGE YOU ARE LOOKING FOR MIGHT HAVE BEEN REMOVED, IT'S NAME CHANGED
      OR ITS TEMPORARILY UNAVAILABLE

please look into this and make the change

Reply:
What changes should be made?